freetextresponse/freetextresponse.py

```diff
@@ -760,6 +759,35 @@ def save_reponse(self, data, suffix=''):
         }
         return result
 
+    @staticmethod
+    def custom_report_format(*args, **kwargs):
+        """
+        This method return the summited answer for the give user and block.
+        """
+        user = kwargs.get('user')
+        block = kwargs.get('block')
+
+        if block and user:
+            student_item_dict = block.student_item_key(user=user)
+            submission = sub_api.get_submissions(student_item_dict, limit=1)
+            try:
+                return submission[0]['answer']
```

Comment:
Could you please do a safe check here for 0 index and 'answer' key? @andrey-canon